### PR TITLE
test(@desktop/chat): Can reply to a message in a community channel

### PR DESCRIPTION
**Pull request by mprakhov**
### What does the PR do

Add a squish test for replying to a msg in the community channel

### Affected areas

Tests in `tst_communityMessageFlows` due to minor refactor

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Looks good. Michal just did a PR that fixes the reply test case on the ChatFlow, in case you want to align to the same way (maybe you already did, I'm too lazy to compare :sweat_smile: ) https://github.com/status-im/status-desktop/pull/8413

**Comment by mprakhov** [comment]
> Looks good. Michal just did a PR that fixes the reply test case on the ChatFlow, in case you want to align to the same way (maybe you already did, I'm too lazy to compare 😅 ) #8413

I'm aligning with the other test case in community, where we pin the message 😄

File: test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py [hunk 1]
```diff
@@ -66,7 +66,12 @@ def step(context, message):
 @When("the user sends multiple test images in the current channel with message \"|any|\" with an image again")
 def step(context, message): 
     _statusCommunityScreen.send_test_image(context.userData["fixtures_root"], True, message)
-    
+
+@When("the user replies to community chat message at index |integer| with \"|any|\"")
+def step(context, message_index: int, message: str):
+    _statusCommunityScreen.toggle_reply_message_at_index(message_index)
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py:72]: lefover! :)

File: test/ui-test/testSuites/suite_communities/tst_communityMessageFlows/test.feature [hunk 2]
```diff
@@ -52,4 +52,13 @@ Feature: Status Desktop community messages
         Then the amount of pinned messages is 2
 
         When the user unpins the message at index 0
-        Then the amount of pinned messages is 1
\ No newline at end of file
+        Then the amount of pinned messages is 1
+
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_communities/tst_communityMessageFlows/test.feature:56]: Leftover!

File: test/ui-test/testSuites/suite_communities/tst_communityMessageFlows/test.feature [hunk 3]
```diff
@@ -52,4 +52,13 @@ Feature: Status Desktop community messages
...
+
+    Scenario Outline: The user can reply to own message
+        Given the user sends a chat message "<message>"
+        When the user replies to community chat message at index 0 with "<reply>"
+        Then the chat message "<reply>" is displayed as a reply of this user's "<message>"
+        Examples:
+     | message                | reply           |
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_communities/tst_communityMessageFlows/test.feature:63]: Some indentation here!!

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 4]
```diff
@@ -315,25 +317,27 @@ def verify_sent_test_image(self, multiple_images: bool, has_message: bool):
             self._verify_image_sent(image_index)
 
 
-    def _do_wait_for_pin_button(self, message_index: int):
+    def _do_wait_for_msg_action_button(self, message_index: int, btn_name: str):
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusCommunityScreen.py:320]: Nice generalization! Thanks! :)

**Review by noeliaSD — APPROVED** [review]
Great! LGTM!

**Pull request merged**